<?xml version="1.0" encoding="UTF-8"?><testrun name="teste" project="teste-de-unidade" tests="10" started="10" failures="1" errors="0" ignored="0">
  <testsuite name="br.com.caelum.leilao.dominio.LeilaoTest" time="0.007">
    <testcase name="naoDeveAceitar2LancesSeguidosMesmoUsuario" classname="br.com.caelum.leilao.dominio.LeilaoTest" time="0.004"/>
    <testcase name="naoDeveAceitar5LancesDoMesmoUsuario" classname="br.com.caelum.leilao.dominio.LeilaoTest" time="0.003">
      <failure>java.lang.AssertionError: expected:&lt;10&gt; but was:&lt;11&gt;
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at br.com.caelum.leilao.dominio.LeilaoTest.naoDeveAceitar5LancesDoMesmoUsuario(LeilaoTest.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
  </testsuite>
  <testsuite name="br.com.caelum.leilao.servico.AvaliadorTest" time="0.009">
    <testcase name="deveEntenderLancesEmOrdemDecrescente" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
    <testcase name="deveEncontrarTresMaioresCom5Lances" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.001"/>
    <testcase name="deveEncontrarTresMaioresVazio" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
    <testcase name="deveEncontrarTresMaioresCom2Lances" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
    <testcase name="deveEntenderLancesEmOrdemCrescente" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
    <testcase name="deveEntenderLeilaoComApenasUmLance" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.008"/>
    <testcase name="deveDevolverValorMedioDosLances" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
    <testcase name="deveEntenderLancesEmOrdemAleatoria" classname="br.com.caelum.leilao.servico.AvaliadorTest" time="0.0"/>
  </testsuite>
</testrun>
